ui/pages/confirm-transaction-base/hardware-connectivity/hardware-connectivity-content.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+import Button from '../../../components/ui/button';
+
+import Typography from '../../../components/ui/typography/typography';
+import {
+  TYPOGRAPHY,
+  FONT_WEIGHT,
+  TEXT_ALIGN,
+} from '../../../helpers/constants/design-system';
+
+export default function HardwareConnectivityContent({
+  deviceName,
+  onConnectClick,
+  onAdvancedClick,
+}) {
+  const t = useI18nContext();
+
+  switch (deviceName) {
+    case 'Ledger':
+      return (
+        <div className="hardware-connectivity-content">
+          <Typography
+            variant={TYPOGRAPHY.H3}
+            fontWeight={FONT_WEIGHT.BOLD}
+            align={TEXT_ALIGN.CENTER}
+          >
+            {t('hardwareWalletConnectivityHelperHeading', [deviceName])}
+          </Typography>
+          <img src="./images/ledger.svg" />
+          <div className="hardware-connectivity-content-list-container">
+            <ol>
+              <li>Plug in your Ledger</li>
+              <li>Enter your passcode</li>
+              <li>Open the Ethereum app</li>
```

Comment:
ToDo:  Use localization for those steps.

Reply:
ToDo: Use localization for those steps.